domo_blueprints/download_dataset_as_csv.py

```diff
@@ -0,0 +1,97 @@
+import sys
+import os
+import argparse
+from pydomo import Domo
+import shipyard_utils as shipyard
+import pandas as pd
+try:
+    import errors as ec
+except BaseException:
+    from . import errors as ec
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--client-id', dest='client_id', required=True)
+    parser.add_argument('--secret-key', dest='secret_key', required=True)
+    parser.add_argument('--dataset-id', dest= 'dataset_id',required = True)
+    parser.add_argument('--destination-file-name',dest = 'dest_file_name',required = True)
+    parser.add_argument('--destination-folder-path',dest = 'dest_folder_path', required = False)
```

Comment:
For consistency across blueprints, `destination-folder-name` and `dest_folder_name`